frontend/src/lib/components/LemonTextArea/index.ts

```diff
@@ -0,0 +1,2 @@
+export { LemonTextArea } from './LemonTextArea'
```

Comment:
I think we're trying to avoid `index.ts` files. According to: https://posthog.com/handbook/engineering/conventions/frontend-coding

Reply:
👍